script.js

```diff
@@ -0,0 +1,41 @@
+const images = document.querySelectorAll('.slider__line img');
+const sliderLine = document.querySelector(' .slider__line');
+let count = 0;
+let width;
+
+function init() {
+    console.log('resize');
+    width = document.querySelector('.slider__block').offsetWidth;
+    sliderLine.style.width = width * images.length + 'px';
+    images.forEach(item => {
+        item.style.width = width + 'px';
+    });
+    rollSlider();
+}
+
+init();
+window.addEventListener('resize', init);
+
+document.querySelector('.slider-next').addEventListener('click', function () {
```

Comment:
А як не знайде елемент? Може краще перестрахуватися?
Ну і загалом функцію краще винести.
```suggestion
const nextClickHandler = () => {/* ... */};
const nextButton = document.querySelector('.slider-next');
if(nextButton !== null) nextButton.addEventListener('click', nextClickHandler);
```